packages/@adobe/spectrum-css-temp/vars/spectrum-global.css

```diff
@@ -451,6 +451,12 @@
   --spectrum-alias-heading-xxs-text-size: var(--spectrum-global-dimension-font-size-100);
   --spectrum-alias-heading6-margin-top: var(--spectrum-global-dimension-font-size-75);
   --spectrum-alias-heading-xxs-margin-top: var(--spectrum-global-dimension-font-size-75);
+  --spectrum-alias-avatar-size-50: var(--spectrum-global-dimension-size-200);
```

Comment:
how'd you decide which ones went in which file?

Reply:
gotcha, it's the newer stuff, i do not understand how that stuff is organized, resolving